tenets/codelingo/go/nil_only_functions/expected.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+   "name": "nil-only-functions",
```

Comment:
Are you using the latest review flow? The result format should be different.

Reply:
will update.